apps/client/ui/Icons/ArrowDown/ArrowDown.tsx

```diff
@@ -2,7 +2,7 @@ import type { FC } from 'react';
 
 import { IconProps } from '@/types/IconType';
 
-const ArrowDownIcon: FC<IconProps> = (props) => (
+const ArrowDownIcon: FC<IconProps> = ({ ...props }) => (
```

Comment:
why this way? 🤔 is it some linter thing?

Reply:
yup - this is the message: Must destructure props in the function signature to initialize an optional prop.eslint[react/require-default-props](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/require-default-props.md)